services/app-api/forms/naaar.json

```diff
@@ -256,12 +256,12 @@
                     {
                       "id": "uITThePQiXntwGGViPTD62",
                       "label": "Specialist",
-                      "hint": "Include all specialists (except for Mental health) within this category. You’ll be able to specify them when creating standards."
+                      "hint": "Include all specialists (except for Mental health) within this category."
```

Comment:
Figma has another sentence here so not sure what is right. same for the next hint